### ISSUE TITLE
All MB sequences got greyed out on Siemens scanner with VE11P

### ISSUE THREAD

**Issue by welton0411** [CMRR-C2P/MB#315]
The MB sequences become unavailable recently on our PETMR scanner with VE11P. We downloaded the lastest MB package [Release R016a for VE11P](https://www.cmrr.umn.edu/multiband/Downloads-Seq/CMRR_MB_EPI_VE11P_R016a.zip) and tried to install on the scanner. However, the warning said "This scanner is almost certainly not the correct baseline for these sequences". The baseline on the scanner is N4_VE11P_LATEST_20161021 and the sequence baseline is N4_VE11P_LATEST_20161021_P4. Even if we installed the package anyway, the scanner is not able to "see" them.

We have updated the latest license and no idea how to proceed. Can anyone shed light?

**Comment by eauerbach** [comment]
Was a service pack or update installed recently on your scanner (_P4)?

**Comment by welton0411** [comment]
No, I don’t think the service pack is installed but I can double check.

WeiTang
